corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -501,18 +519,48 @@ <h4 class="modal-title">{% trans "Get a Demo of CommCare" %}</h4>
               </div>
             </div>
           </div>
+
+
+          <div class="modal fade"
+               id="cta-form-get-demo-new">
+            <div class="modal-dialog">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                  <h4 class="modal-title">{% trans "Request a CommCare Demo" %}</h4>
+                </div>
+                <div class="modal-body form-horizontal">
+
+                  {# Hubspot Form #}
+                  <div class="form form-horizontal"
+                           id="get-demo-cta-form-content"></div>
+
+                  {# Schedule Once Demo #}
+                  <div id="get-demo-cta-calendar-content" style="display: none;">
+                    <p>
+                      {% blocktrans %}
+                        Select a time to see CommCare in action and review
+                        questions with our team.
+                      {% endblocktrans %}
+                    </p>
+                    <div id="SOIDIV_commcaredemoform"
+                         data-so-page="commcaredemoform"
+                         data-style="border: 1px solid #d8d8d8; min-width: 290px; max-width: 900px;"
+                         data-psz="11"></div>
+                  </div>
+
+                </div>
+              </div>
+            </div>
+          </div>
+
           <!--[if lte IE 8]>
           <script charset="utf-8" src="//js.hsforms.net/forms/v2-legacy.js"></script>
           <![endif]-->
           <script charset="utf-8" src="//js.hsforms.net/forms/v2.js"></script>
-          <script>
-            hbspt.forms.create({
-              portalId: "{{ ANALYTICS_IDS.HUBSPOT_API_ID }}",
-              formId: "0f5de42e-b562-4ece-85e5-cfd2db97eba8",
-              target: "#get-demo-cta-form-body",
-              css: ""
-            });
-          </script>
+
+          <script src="{% static 'analytix/js/hubspot.get_demo.js' %}"></script>
```

Comment:
This script tag being here will likely break requirejs; I'm guessing it will end up going into [analytics_js.html](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/analytics/templates/analytics/analytics_js.html)  once you make the IE8 checking and fetching of `v2.js` into javascript.

Reply:
resolved by https://github.com/dimagi/commcare-hq/pull/22503